app/assets/javascripts/utils/course_utils.js

```diff
@@ -117,22 +112,22 @@ const CourseUtils = class {
     return `https://${language}.${project}.org/wiki/${underscoredTitle}`;
   }
 
-  formattedArticleTitle(language, project, articleTitle) {
+  formattedArticleTitle(article, defaultWiki) {
     let languagePrefix = '';
-    if (language === undefined || language === 'en') {
+    if (defaultWiki === undefined || article.language === defaultWiki.language || article.language === null || article.language === undefined) {
```

Comment:
These can be simplified somewhat, I think. Instead of checking article.language and defaultWiki against multiple precise conditions, we can just check for falsiness: `!defaultWiki` and `!article.language`.